Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -230,12 +241,19 @@ void MemChecks::AddFromStrings(const TMemChecksStr& mc_strings)
     mc.is_break_on_write = flags.find('w') != flags.npos;
     mc.log_on_hit = flags.find('l') != flags.npos;
     mc.break_on_hit = flags.find('b') != flags.npos;
+    if (flags.find('c') != std::string::npos)
+    {
+      iss >> std::ws;
+      std::string condition;
+      std::getline(iss, condition);
+      mc.condition = Expression::TryParse(condition);
+    }
```

Comment:
Shouldn't raw lines (using `$`) be used? Saving/loading memory BPs don't seem to work properly when there is a equal sign in the condition.

Reply:
I just copied it over from the original BP code. I'm not sure why it works for breakpoints and not memory BPs.  How do I fix it with using raw lines here? Not too keen on loading strings.

Also the failure is in the ini.GetLines() function. It fails to return anything.

Reply:
Nevermind, I finally won spot the difference.